rust/fuzz/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Interesting.
Why do we need specific rust fuzz targets ?
Cannot we use the C wrappers ?